tools/js-sdk-release-tools/src/test/tspLocation/updateTspLocation.test.ts

```diff
@@ -0,0 +1,16 @@
+import * as fs from "fs";
```

Comment:
merge this test with the one that updates user agent with in the same `describe('update generated SDK')`